src/components/OSCALControl.js

```diff
@@ -85,14 +85,14 @@ export default function OSCALControl(props) {
               control={props.control}
               controlId={props.control.id}
               parameters={props.control.params}
-              implReqStatements={props.implReqStatements}
```

Comment:
For this and similar changes, `implementedRequirement` contains `implReqStatements` so we don't need to pass around both.